### ISSUE TITLE
Invisible Spray wearing off after folding a body bag

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#29496]
(WEB REPORT BY: sugarbone REMOTE: 206.221.180.138:7777)
# Revision
648176ce0040d63f21f845a43f922bfbca856fb3

# Description
The invisible spray wears off if a body bag is folded
# Steps to Reproduce
1. Spray a body bag with invisible spray
2. Fold the closed and invisible into your hand
3. Observe it becoming visible again
# What you Expected
I expected it to stay invisible no matter how many times I fold and unfold it
# What Actually Happened
The invisible effect wore off after I folded the body bag into my hand

**Comment by robintime** [comment]
i thiiink i know why this happens

iirc bodybags in the code are closets when you deploy them, and when you fold them up it deletes the closet and spawns the folded bodybag item. the issue here is that there's nothing that makes the folded bodybag inherit/transfer things like invisispray to its other state.

(or i could be wrong)

**Comment by GreatBigFailure** [comment]
We could really benefit from more crafting/transferring item helper procs. Like something to transfer invis, glue, etc. Something to check contents of an item about to be destroyed during crafting. Something to just destroy the old item and create the new one. Would fix this and make making craftable things easier.

**Comment by Eneocho** [comment]
Any item general update removes the invisible spray. Say you sprayed your Swiss army knife for some fucking reason. When you update it to get another tool you lose the invisispray.

**Comment by west3436** [comment]
Yes this also happens to placed invisible C4, would be a great thing to fix generally.
